Sources/Proxy/Core/ViewProxy/index.js

```diff
@@ -257,8 +257,12 @@ function vtkViewProxy(publicAPI, model) {
 
   // --------------------------------------------------------------------------
 
+  publicAPI.lightResetCamera = model.renderer.resetCamera;
+
+  // --------------------------------------------------------------------------
+
   publicAPI.resetCamera = () => {
-    model.renderer.resetCamera();
+    publicAPI.lightResetCamera();
     model.renderer.resetCameraClippingRange();
```

Comment:
is it still needed to call `resetCameraClippingRange()` when `lightResetCamera()` does also reset clipping ranges ?